package.json

```diff
@@ -103,7 +103,7 @@
     "babel-plugin-transform-rename-import": "2.3.0",
     "babel-plugin-typescript-to-proptypes": "1.2.1",
     "cross-env": "7.0.0",
-    "cypress": "4.0.1",
+    "cypress": "3.8.3",
```

Comment:
Needs to be done separately I think. The GitHub action and the asset upload don't seem in sync as paths may have changed. Just a guess so far. Still, better isolate this I think.